src/redist/targets/GenerateBundledVersions.targets

```diff
@@ -266,8 +266,23 @@
 
       <Crossgen2SupportedRids Include="@(Net60Crossgen2SupportedRids)" />
 
-      <!-- Match the Crossgen2 RIDs although some are currently not supported in NativeAOT. Its better for the SDK not to block on the RIDs and let NativeAOT manage the experience -->
-      <ILCompilerSupportedRids Include="@(Net60Crossgen2SupportedRids)" />
+      <Net70ILCompilerSupportedRids Include="
+        linux-arm64;
+        linux-musl-x64;
```

Comment:
Should linux-musl-arm64 be in this list instead?

We are publishing a package for it in .NET 7: https://www.nuget.org/packages/runtime.linux-musl-arm64.Microsoft.DotNet.ILCompiler/7.0.5